components/automate-cli/pkg/client/apiclient/apiclient.go

```diff
@@ -10,7 +10,6 @@ import (
 	client_type "github.com/chef/automate/components/automate-cli/pkg/client"
```

Comment:
automate-cli failed the build with a mysterious message:
```
cannot load github.com/chef/automate/components/automate-gateway/api/: no Go source files
```
Tracked that down to this handful of files needing the old path removed (introspection is now under automate-gateway/api/iam/v2)